File: common/src/main/java/com/ssblur/alchimiae/events/ClientQuitEvent.java
package com.ssblur.alchimiae.events;

import com.ssblur.alchimiae.alchemy.ClientAlchemyHelper;
import dev.architectury.event.events.client.ClientPlayerEvent;
import net.minecraft.client.player.LocalPlayer;

public class ClientQuitEvent implements ClientPlayerEvent.ClientPlayerQuit {
  @Override
  public void quit(LocalPlayer player) {
    ClientAlchemyHelper.reset();
  }
}

File: common/src/main/kotlin/com/ssblur/alchimiae/events/ClientQuitEvent.kt
package com.ssblur.alchimiae.events

import com.ssblur.alchimiae.alchemy.ClientAlchemyHelper
import dev.architectury.event.events.client.ClientPlayerEvent
import net.minecraft.client.player.LocalPlayer

class ClientQuitEvent : ClientPlayerEvent.ClientPlayerQuit {
  override fun quit(player: LocalPlayer?) {
    ClientAlchemyHelper.reset()
  }
}
